.vscode/launch.json

```diff
@@ -3,63 +3,37 @@
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
-  "configurations": [
+  "inputs": [
     {
-      "name": "Launch test/e2e/app-dir/app development",
```

Comment:
Can we keep these by composing them with the new generic commands?

iirc VSCode would show a list of recently used commands so you're potentially saving a keystroke by just executing the recent command. Without the specific command you'd always have to click multiple times. That's especially annyoing if you want an option that's not the first in the selection.

Reply:
@eps1lon Just to visualise what I wrote here, this is how the workflow would be for triggering the last used launch config.

https://github.com/vercel/next.js/assets/761683/54871ed2-28fb-4578-846a-d4c07d6bd0af

Reply:
> I'm a bit hesitant to re-introduce the noisy duplication into the launch.json, it felt so neat with the inputs. 

Function over form.

> But since in this case you need to edit the launch.json to customise the app dirname anyways

Why anyway? Was the original launch config not working as-is?

Reply:
> > I'm a bit hesitant to re-introduce the noisy duplication into the launch.json, it felt so neat with the inputs.
> 
> Function over form. The muscle memory of devs is way more valuable then code asthethics.

I think you're right.

> > But since in this case you need to edit the launch.json to customise the app dirname anyways
> 
> Why anyway? Was the original launch config not working as-is?

It's working as-is if you want to run `test/e2e/app-dir/app`, otherwise you need to edit the dirname.

I will restore the original configs...